pyqt-configure

```diff
@@ -1,18 +1,25 @@
 #!/bin/sh
 
 
-PYTHON=python2
+PYTHON=python3
 PYTHON_VERSION=$(${PYTHON} -c 'import sys; print(sys.version[:3])')
 
-
 ${PYTHON} configure.py \
+    --assume-shared \
     --confirm-license \
     --enable=QtCore \
     --enable=QtGui \
     --enable=QtNetwork \
+    --enable=QtWidgets \
     --no-designer-plugin \
     --no-docstrings \
+    --no-qml-plugin \
     --no-qsci-api \
+    --sysroot=/app \
+    -w \
```

Comment:
This is the same as `--verbose`, so don't add both.

The build is already quite verbose though, so to be honest I'd rather remove them both. (but if we do keep one, let's keep `--verbose` only)